src/views/portfolio/sidebar/index.tsx

```diff
@@ -476,12 +520,23 @@ const RSR = () => {
     decimals: 18,
   }
 
+  const sortedChainIds = useMemo(() => {
+    if (!rsrBalances) return []
+    return Object.keys(RSR_ADDRESS)
+      .map(Number)
+      .sort((a, b) => {
+        const aBalance = (rsrBalances[a] as bigint) ?? 0n
+        const bBalance = (rsrBalances[b] as bigint) ?? 0n
+        return aBalance > bBalance ? -1 : aBalance < bBalance ? 1 : 0
```

Comment:
why `as bigint` and not `BigInt(x)` ?
Why is it required for them to be a big ints if we are only comparing 2 numbers